### ISSUE TITLE
This plugin changed my life

### ISSUE THREAD

**Issue by CPlusPatch** [Bobrobot1/EnmityLinuxify#3]
Two years ago I was severely accosted by life's many aches and pains. I was searching for an answer. I looked f*ckin everywhere, man. Then I stumbled upon this plugin while looking for a good hardy background plugin while I ate my pasta (elbow noodles with butter and half a dash of salt). I scoffed at the vapid, self insistent stupid fucking red faced README at first. But then I used the plugin. Ten seconds in I TRANSCENDED. I didn't know what to call what I experienced, but while using the the plugin (sobbing LOUDLY), I researched what it could possibly be. I was FUCKING TERRIFIED that this feeling might be too big for ANY kind of word to define it. But then, scrolling through Reddit, I found r/DMT. These brave souls on this page had THE WORD of all words to describe EXACTLY what I read in the first ten seconds of this melodious README. The term was EGO DEATH. That's right, Ego death. Look it up. It means a complete LOSS of self identity. Yep, I lost myself in that moment. And I was reborn as this monkey. At first I hated that damn monkey, but now I realized I was that monkey all along. I AM THE ONE SOAKING IN THE WATER THAT IS LIFE!1!!! 

After reading all this I hope you learned something. I don't need to spell it out for you but I will anyways (your welcome): Just who exactly do you fucking think is behind all this? All these corporations, and all these governments, all this CHAOS.... 

Its the pope. 

Look around.

TLDR: Funny monkey takes a swim, hijinks ensue
